relay-general/src/store/normalize/breakdowns.rs

```diff
@@ -82,6 +84,37 @@ fn get_op_time_spent(mut intervals: Vec<TimeWindowSpan>) -> Option<f64> {
     Some(op_time_spent)
 }
 
+fn span_from_trace_context(event: &Event) -> Option<Annotated<Span>> {
```

Comment:
Instead of doing this I would create a trait that abstracts over spans and trace contexts, called `SpanLike`. That's probably useful across the codebase.